src/renderer/runtime.ts

```diff
@@ -709,9 +723,13 @@ export class Runtime {
    */
   viewerToWorld(x: number, y: number) {
     const scaleFactor = this.getScaleFactor();
-    this._viewerToWorld.x = this.target[1] + x / scaleFactor;
-    this._viewerToWorld.y = this.target[2] + y / scaleFactor;
-    return this._viewerToWorld;
+    // is this right?
+    const xo = this.target[1] + x / scaleFactor;
```

Comment:
part of me expected the following:
```
    const xo = (this.target[1] + x) / scaleFactor;
```
but I don't think it's right

Reply:
I think the linter removed the parenthesis at some point, but it is right, it should be:
```ts
const xo = this.target[1] + (x / scaleFactor);
```
Since the target is already in world-space and the x is what needs to be scaled